go/vt/vtgate/executor_vschema_ddl_test.go

```diff
@@ -541,171 +504,112 @@ func TestExecutorAddDropVindexDDL(t *testing.T) {
+			require.Fail(t, "timed out waiting for test_lookup vindex to be removed")
 		}
 		time.Sleep(1 * time.Millisecond)
 	}
 
 	// use the newly created vindex on a new table
 	stmt = "alter vschema on test2 add vindex test_hash (id)"
 	_, err = executor.Execute(context.Background(), "TestExecute", session, stmt, nil)
-	if err != nil {
-		t.Fatalf("error in %s: %v", stmt, err)
-	}
+	require.NoError(t, err)
 
 	vschema, vindex = waitForVindex(t, ks, "test_hash", vschemaUpdates, executor)
-	if vindex.Type != "hash" {
-		t.Errorf("vindex type %s not hash", vindex.Type)
-	}
+	require.Equal(t, "hash", vindex.Type)
 
-	if table, ok := vschema.Keyspaces[ks].Tables["test2"]; ok {
-		if len(table.ColumnVindexes) != 1 {
-			t.Fatalf("table vindexes want 1 got %d", len(table.ColumnVindexes))
-		}
-		if table.ColumnVindexes[0].Name != "test_hash" {
-			t.Fatalf("table vindexes didn't contain test_hash")
-		}
-	} else {
-		t.Fatalf("table test2 not defined in vschema")
-	}
+	table, ok := vschema.Keyspaces[ks].Tables["test2"]
+	require.Truef(t, ok, "table test2 not defined in vschema")
+	require.Len(t, table.ColumnVindexes, 1)
+	require.Equal(t, "test_hash", table.ColumnVindexes[0].Name)
 
 	// create an identical vindex definition on a different table
 	stmt = "alter vschema on test2 add vindex test_lookup (c1,c2) using lookup with owner=`test`, from=`c1,c2`, table=test_lookup, to=keyspace_id"
 	_, err = executor.Execute(context.Background(), "TestExecute", session, stmt, nil)
-	if err != nil {
-		t.Fatalf("error in %s: %v", stmt, err)
-	}
+	require.NoError(t, err)
 
 	vschema, vindex = waitForVindex(t, ks, "test_lookup", vschemaUpdates, executor)
-	if vindex.Type != "lookup" {
-		t.Errorf("vindex type %s not hash", vindex.Type)
-	}
+	require.Equal(t, "lookup", vindex.Type)
 
-	if table, ok := vschema.Keyspaces[ks].Tables["test2"]; ok {
-		if len(table.ColumnVindexes) != 2 {
-			t.Fatalf("table vindexes want 1 got %d", len(table.ColumnVindexes))
-		}
-		if table.ColumnVindexes[1].Name != "test_lookup" {
-			t.Fatalf("table vindexes didn't contain test_lookup")
-		}
-	} else {
-		t.Fatalf("table test2 not defined in vschema")
-	}
+	table, ok = vschema.Keyspaces[ks].Tables["test2"]
+	require.Truef(t, ok, "table test2 not defined in vschema")
+	require.Len(t, table.ColumnVindexes, 2)
+	require.Equal(t, "test_lookup", table.ColumnVindexes[1].Name)
 
 	qr, err = executor.Execute(context.Background(), "TestExecute", session, "show vschema vindexes on TestExecutor.test2", nil)
-	if err != nil {
-		t.Fatalf("error in show vschema vindexes on TestExecutor.test2: %v", err)
-	}
+	require.NoError(t, err)
 	wantqr = &sqltypes.Result{
 		Fields: buildVarCharFields("Columns", "Name", "Type", "Params", "Owner"),
 		Rows: [][]sqltypes.Value{
 			buildVarCharRow("id", "test_hash", "hash", "", ""),
 			buildVarCharRow("c1, c2", "test_lookup", "lookup", "from=c1,c2; table=test_lookup; to=keyspace_id", "test"),
 		},
-		RowsAffected: 2,
```

Comment:
Same here

Reply:
select / show queries does not have any value set for RowsAffected

Reply:
they only make sense for DMLs

Reply:
How was the test working previously then?

Reply:
there was change where we stopped setting rowsAffected for non-dml queries.
But, this test was always in skip from along time, hence it was never changed (fixed).